CoreScriptsRoot/Modules/DeveloperConsoleModule.lua

```diff
@@ -2258,7 +2327,11 @@ end
 ----------------
 -- Scroll bar --
 ----------------
-function Methods.ApplyScrollbarToFrame(devConsole, scrollbar, window, body, frame)
+function Methods.ApplyScrollbarToFrame(devConsole, 
```

Comment:
If so, is there a reason we don't use ScrollingFrame? Some bug or defect or feature missing?